drivers/dma/Kconfig

```diff
@@ -34,6 +34,12 @@ config DMA_2_NAME
 	help
 	  Device name for DMA Controller 2.
 
+config DMA_64BIT
```

Comment:
As this is only supported on one platform, should it have a depends HAS_DMA_64BIT so the option doesn't show on other architectures?

Reply:
ack

Reply:
If Kconfig works like linux Kconfig that "DMA 64 address support" string should be removed.  Could lineup names to linux kernel/dma/Kconfig HAS_DMA and ARCH_DMA_ADDR_T_64BIT?